### ISSUE TITLE
Foundry v12 is not supported

### ISSUE THREAD

**Issue by sakulstra** [fryguy1013/pdftofoundry#17]
Hello, I just tried this module and first of - it's awesome!

That said, i faced some issues with chapter 2 and 6 of extinction curse.
On both i see sth like:
```
Error: ScenePF2e validation errors:
  walls: 
    LBOgFOT7FixeUjYd: 
      move: 1 is not a valid choice
      sound: 1 is not a valid choice
    hJ2nZuQCEeI7EW7o: 
      move: 1 is not a valid choice
      sound: 1 is not a valid choice
    xiSda1DXsuG3BHuR: 
      move: 1 is not a valid choice
      sound: 1 is not a valid choice
    E3QxLHqKiQrJpUGo: 
      move: 1 is not a valid choice
      sound: 1 is not a valid choice
```
in my console.

Using foundry 12, which might be part of the problem, but rest has worked just fine.

**Comment by fryguy1013** [comment]
Yeah, probably because it doesn't support v12.

**Comment by sakulstra** [comment]
@fryguy1013 yes, checked and i could import on v11 and then upgrade to 12 which was an okayish workaround.

Is the code of this module open source somewhere? Wanted to check if i could help/understand how this actually works as it seems quite magic to me 😅  but here on github i could only find the minified source.

**Comment by draconswan** [comment]
Is there a planned update for this module?

**Comment by SomeOctopus** [comment]
> Yeah, probably because it doesn't support v12.

Count me in as also interested in whether the module will be updated to v12.

**Comment by Afgar** [comment]
@fryguy1013 Are you working on updating this for version 12? I updated my PF2E to the new version without backing up and now I don't have a version of both this and PF2E that can be ran on the same Foundry version. Making the downgrade to v11 and then back up to 12 not possible.

**Comment by ryeako** [comment]
I'm finding similar is with import with Agents of Edgewatch book 2.  Mine only came up with wall errors and failed to import any of the maps.  All the journal entries and NPCs came in but several handouts failed.

**Comment by Olliebird** [comment]
@fryguy1013 Are there any plans to update to v12? Several v12 issues have been closed as "not planned".

**Comment by JamesSkemp** [comment]
FYI, been holding off on upgrading to 12 because of this module, and after looking around elsewhere for news:

> [TMun357](https://www.reddit.com/user/TMun357/)
•
[3d ago](https://www.reddit.com/r/FoundryVTT/comments/1eseih7/comment/li5op1e/)
PF2e System Developer
I can tell you he is working on making it V12 compatible, but it is taking time because the walling is an issue. It may be unsolvable but it is being worked on :)
> 
> No ETA so be pleasantly surprised…

From https://www.reddit.com/r/FoundryVTT/comments/1eseih7/comment/li5op1e/

**Comment by fryguy1013** [comment]
Fixed in 5.0.0
